package.json

```diff
@@ -52,10 +52,7 @@
     "minilog": "2.0.8",
     "node-sass": "3.3.3",
     "po2icu": "git://github.com/LLK/po2icu.git#develop",
-    "react": "0.14.0",
-    "react-addons-test-utils": "0.14.0",
-    "react-dom": "0.14.0",
-    "react-intl": "2.0.0-beta-1",
```

Comment:
Correct. These aren't used in our current code as we include them in `./static/js/lib`.